### PR TITLE
Registration delivery:- flutter component updrade

### PR DESCRIPTION
**Pull request by yashita-egov**

### CODE REVIEW

File: packages/registration_delivery/lib/pages/reason_for_deletion.dart [hunk 1]
```diff
@@ -115,68 +124,58 @@ class ReasonForDeletionPageState extends LocalizedState<ReasonForDeletionPage> {
                         // context.router.push(HouseholdAcknowledgementRoute(
```
Comment (naveenr-egov) [inline packages/registration_delivery/lib/pages/reason_for_deletion.dart:124]: Remove the commented code

File: packages/registration_delivery/lib/widgets/action_card/action_card.dart
Comment (naveenr-egov) [inline packages/registration_delivery/lib/widgets/action_card/action_card.dart:1]: Why is this deleted?

File: packages/registration_delivery/lib/widgets/inventory/no_facilities_assigned_dialog.dart [hunk 2]
```diff
@@ -1,27 +1,34 @@
...
+      context: context,
+      builder: (ctx)=> Popup(
         titleIcon: Icon(
           Icons.warning,
-          color: Theme.of(context).colorScheme.error,
+          color: DigitTheme.instance.colors.light.alertError,
         ),
-        titleText: 'No facilities assigned',
-        contentText: 'Please select another boundary or '
+        title: 'No facilities assigned',
```
Comment (naveenr-egov) [inline packages/registration_delivery/lib/widgets/inventory/no_facilities_assigned_dialog.dart:17]: Use localization codes. Check for existing codes for both title and description

File: packages/registration_delivery/lib/widgets/progress_bar/beneficiary_progress.dart [hunk 3]
```diff
@@ -84,12 +85,13 @@ class BeneficiaryProgressBarState extends State<BeneficiaryProgressBar> {
     final target = targetModel?.targetNo ?? 0.0;
 
     return DigitCard(
-      child: ProgressIndicatorContainer(
+      margin: const EdgeInsets.all(spacer2),
```
Comment (naveenr-egov) [inline packages/registration_delivery/lib/widgets/progress_bar/beneficiary_progress.dart:88]: Why is the margin added?

File: packages/registration_delivery/lib/widgets/table_card/table_card.dart
Comment (naveenr-egov) [inline packages/registration_delivery/lib/widgets/table_card/table_card.dart:1]: Why is this added in registration_delivery?

**Review by naveenr-egov — CHANGES_REQUESTED** [review]
Resolve comments.

Please format the files to avoid spacing changes. User dart format before committing. This is applicable to all files.

File: packages/registration_delivery/lib/widgets/action_card/action_card.dart
Comment (yashita-egov) [reply packages/registration_delivery/lib/widgets/action_card/action_card.dart:1]: same component coming from digit ui library.

**Review by naveen-egov — CHANGES_REQUESTED** [review]
please u

File: packages/registration_delivery/lib/widgets/progress_bar/beneficiary_progress.dart (hunk 3 above)
Comment (yashita-egov) [reply packages/registration_delivery/lib/widgets/progress_bar/beneficiary_progress.dart:88]: by default no margin is present in new digit component (digit card)

File: packages/registration_delivery/lib/pages/beneficiary/beneficiary_checklist.dart [hunk 4]
```diff
@@ -397,29 +420,17 @@ class _BeneficiaryChecklistPageState
...
-                                          builder: (context, state) {
-                                            return SelectionBox<bool>(
-                                              //label: e,
+                                    children: [
+                                      BlocBuilder<ServiceBloc, ServiceState>(
+                                        builder: (context, state) {
+                                          return Align(
+                                            alignment: Alignment.topLeft,
+                                            child: SelectionCard<bool>(
+                                              // label: e,
```
[the remark this answers is not in the document]
Comment (yashita-egov) [reply packages/registration_delivery/lib/pages/beneficiary/beneficiary_checklist.dart:429]: was already commented

File: packages/registration_delivery/lib/pages/beneficiary/refer_beneficiary.dart [hunk 5]
```diff
@@ -98,20 +101,21 @@ class ReferBeneficiaryPageState extends LocalizedState<ReferBeneficiaryPage> {
...
+                    return Button(
+                      label: localizations.translate(i18.common.coreCommonSubmit),
+                      type: ButtonType.primary,
+                      size: ButtonSize.large,
+                      mainAxisSize: MainAxisSize.max,
+                      isDisabled: isClicked?true:false,
+                      onPressed: () {
                               if (form.control(_referralReason).value == null) {
                                 clickedStatus.value = false;
-                                form
```
[the remark this answers is not in the document]
Comment (yashita-egov) [reply packages/registration_delivery/lib/pages/beneficiary/refer_beneficiary.dart:112]: error handling already done by the component

File: packages/registration_delivery/lib/widgets/table_card/table_card.dart
Comment (yashita-egov) [reply packages/registration_delivery/lib/widgets/table_card/table_card.dart:1]: no such widget in digit ui library.

**Comment by naveenr-egov** [comment]
Merge with this https://github.com/egovernments/health-campaign-field-worker-app/pull/609
